sys/net/gnrc/Makefile.dep

```diff
@@ -172,6 +172,14 @@ ifneq (,$(filter gnrc_netif_bus,$(USEMODULE)))
   USEMODULE += core_msg_bus
 endif
 
+ifneq (,$(filter netdev_eth slipdev, $(USEMODULE)))
```

Comment:
Would that cover esp32 or is the esp32 using its own IP implementation anyway?

Reply:
Both `esp_wifi` and `esp_eth` will select `netdev_eth`.
`esp_now` is a 6lo interface. 

And ESP does use our GNRC stack or LWIP, just like everyone else.